plugins/scaffolder/src/components/fields/EntityPicker/EntityPicker.tsx

```diff
@@ -36,6 +36,8 @@ export const EntityPicker = ({
 }: FieldProps<string>) => {
   const allowedKinds = uiSchema['ui:options']?.allowedKinds as string[];
   const defaultKind = uiSchema['ui:options']?.defaultKind as string | undefined;
+  const allowArbitraryValues = (uiSchema['ui:options']?.allowArbitraryValues ||
```

Comment:
Does this ever resolve to false? The `||` is always going to just evaluate true right?

Reply:
🤦🏽 you're right, great catch.

Reply:
I forced pushed a correction, as well as some unit tests